features/behat-steps.feature

```diff
@@ -0,0 +1,741 @@
+
+  @require-wp
+  Scenario: Test WP installation in subdirectory
+    Given a WP installation in 'subdir'
+    When I run `wp core version` from 'subdir'
+    Then STDOUT should not be empty
+    And the return code should be 0
+
+  @require-wp
+  Scenario: Test plugin installation step
+    Given a WP installation
+    And these installed and active plugins:
+      """
+      hello
+      """
+    When I run `wp plugin list --status=active --field=name`
+    Then STDOUT should contain:
+      """
+      hello
+      """
+
+  @require-wp
+  Scenario: Test version string comparison
+    Given a WP installation
+    When I run `wp core version`
+    Then STDOUT should be a version string >= 4.0
+
+  @require-wp
+  Scenario: Test STDOUT as table containing rows
+    Given a WP installation
+    When I run `wp option list --fields=option_name --format=table --orderby=option_name`
+    Then STDOUT should be a table containing rows:
+      | option_name |
+
+  @require-wp
+  Scenario: Test JSON output
+    Given a WP installation
+    When I run `wp option get siteurl --format=json`
+    Then STDOUT should be JSON containing:
+      """
+      "example.com"
+      """
+
+  @require-wp
+  Scenario: Test CSV output
+    Given a WP installation
+    When I run `wp user list --format=csv --fields=user_login`
+    Then STDOUT should contain:
+      """
+      user_login
+      """
+
+  @require-wp
+  Scenario: Test YAML output
+    Given a WP installation
+    When I run `wp cli info --format=yaml`
+    Then STDOUT should be YAML containing:
+      """
+      PHP binary:
+      """
+
+  @require-wp
+  Scenario: Test save file as variable
+    Given a WP installation
+    And a composer.json file:
+      """
+      {"name": "test"}
+      """
+    And save the {RUN_DIR}/composer.json file as {COMPOSER}
+    When I run `echo '{COMPOSER}'`
+    Then STDOUT should contain:
+      """
+      test
+      """
+
+  Scenario: Test HTTP request mocking
+    Given that HTTP requests to https://example.com/test will respond with:
+      """
+      HTTP/1.1 200
+      Content-Type: text/plain
+
+      Mock response
+      """
+    Then the wp-cli.yml file should exist
+    And the mock-requests.php file should exist
+
+  @require-wp
+  Scenario: Test background process launch
+    Given a WP installation
+    When I launch in the background `wp eval 'sleep(2); echo "done";'`
+    # Background process should not block, continuing immediately
+
+  @require-wp
+  Scenario: Test custom wp-content directory
+    Given a WP installation
+    And a custom wp-content directory
+    Then the my-content directory should exist
+    And the my-plugins directory should exist
+    And the my-mu-plugins directory should exist
+    And the wp-config.php file should contain:
+      """
+      WP_CONTENT_DIR
+      """
+
+  @require-wp
+  Scenario: Test WP multisite subdirectory installation
+    Given a WP multisite subdirectory installation
+    When I run `wp site list --field=url`
```

Comment:
Use `wp core is-installed --network` instead to verify that it's a multisite

`wp site` is not available in this repo.